x/paychan/handler_test.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/iov-one/weave/x/cash"
 )
 
+var now = time.Now().UTC()
```

Comment:
I was about to complain about use of time.Now(), then saw it was only in test code.  Good